Jenkinsfile.conda

```diff
@@ -1,98 +1,72 @@
+                        --env TS_CONFIG_OCS_DIR=/home/saluser/ts_config_otcs \
+                        --env LSST_DDS_DOMAIN=citest \
+                        -v ${WORKSPACE}:/home/saluser/source ${dockerImageName}
                     """
                 }
             }
         }
-        stage("Create Conda Dev package") {
-            when {
-                not {
-                    buildingTag()
+        stage("Clone ts_config_ocs"){
+            steps {
+                script {
+                    sh """
+                    docker exec ${container_name} sh -c "git clone https://github.com/lsst-ts/ts_config_ocs.git"
+                    """
                 }
             }
+        }
+        stage("Create Conda package") {
             steps {
-                withEnv(["HOME=${env.WORKSPACE}"]) {
+                script {
                     sh """
-                        cd ${HOME}/conda
-                        source /home/saluser/miniconda3/bin/activate
-                        conda config --add channels conda-forge
-                        conda config --add channels lsstts
-                        source ${OSPL_HOME}/release.com
-                        conda build -c lsstts/label/dev --prefix-length 100 .
+                    docker exec ${container_name} sh -c 'cd ~/source/conda && source ~/miniconda3/bin/activate && source "\$OSPL_HOME/release.com" && conda build --prefix-length 100 .'
                     """
                 }
             }
         }
-
-        stage("Push Conda Release package") {
-            when {
-                buildingTag()
-            }
+        stage("Push Conda package") {
             steps {
                 withCredentials([usernamePassword(credentialsId: 'CondaForge', passwordVariable: 'anaconda_pass', usernameVariable: 'anaconda_user')]) {
-                    withEnv(["HOME=${env.WORKSPACE}"]) {
+                    script {
                         sh """
-                        source /home/saluser/miniconda3/bin/activate
-                        anaconda login --user ${anaconda_user} --password ${anaconda_pass}
-                        anaconda upload -u lsstts --force /home/saluser/miniconda3/conda-bld/linux-64/ts-${package_name}*.tar.bz2
+                        docker exec ${container_name} sh -c "source ~/miniconda3/bin/activate && \
+                            anaconda login --user ${anaconda_user} --password ${anaconda_pass} && \
+                            anaconda upload -u lsstts --force \
+                            ~/miniconda3/conda-bld/linux-64/ts-${package_name}*.tar.bz2"
                         """
                     }
                 }
             }
         }
     }
     post {
-        always {
-            withEnv(["HOME=${env.WORKSPACE}"]) {
-                sh 'chown -R 1003:1003 ${HOME}/'
-            }
-        }
         cleanup {
-            deleteDir()
+            sh """
+            docker stop ${container_name}
```

Comment:
Replacing docker command with build-in docker support will remove need for docker stop..